cmd/tempo/app/modules.go

```diff
@@ -156,24 +158,27 @@ func (t *App) initQuerier() (services.Service, error) {
 
 func (t *App) initQueryFrontend() (services.Service, error) {
 	var err error
-	t.frontend, err = cortex_frontend.New(t.cfg.Frontend.Config, util.Logger, prometheus.DefaultRegisterer)
+
+	originalTripper, v1, _, err := cortex_frontend.InitFrontend(t.cfg.Frontend.Config, frontend.CortexNoQuerierLimits{}, 0, util.Logger, prometheus.DefaultRegisterer)
```

Comment:
non blocking nit: `originalTripper` sounds weird, can we use `cortexFrontendTripper`? `grpcTripper`?

Reply:
Updated